doc-content/enterprise-only/project-data/modify-projects-overview-con.adoc

```diff
@@ -3,4 +3,4 @@
 
 {CENTRAL} includes a number of sample projects that you can use to get familiar with the product and its features. The sample projects have been designed and created to demonstrate a variety of business scenarios. You can modify the sample projects to meet your specific business needs. For example, {PRODUCT} {PRODUCT_VERSION} contains the *Mortgage Process* sample project, which consists of predefined data objects, guided decision tables, guided rules, forms, and a business process. You can edit the sample to refine your business process.
 
-If none of the existing {CENTRAL} project samples align with your requirements, you can import a new project from a git repository. See xref:git-import-project[].
+If none of the existing {CENTRAL} project samples align with your requirements, you can create a new project or import one from a git repository. See xref:git-import-project[].
```

Comment:
This phrase seems to mean that importing a project from e git repo is for samples. But importing a project from a git repository is not limited to samples.  A project developed in another BC instance can definitely also be imported, and perhaps also some projects created elsewhere.

Reply:
Corrected.